src/rust/ide/src/controller/module.rs

```diff
@@ -68,14 +97,28 @@ impl Path {
         // The file stem existence should be checked during construction.
         self.file_path.file_stem().unwrap()
     }
+
     /// Create a module path consisting of a single segment, based on a given module name.
-    #[cfg(test)]
-    pub fn from_module_name(name:impl Str) -> Self {
+    /// The `default` is used
+    pub fn from_mock_module_name(name:impl Str) -> Self {
         let name:String = name.into();
         let file_name   = format!("{}.{}",name,constants::LANGUAGE_FILE_EXTENSION);
         let file_path   = FilePath::new(default(),&[file_name]);
         Self::from_file_path(file_path).unwrap()
     }
+
+    /// Module's qualified name is used in some of the Language Server's APIs, like
+    /// `VisualisationConfiguration`.
+    ///
+    /// Qualified name is constructed as follows:
+    /// `ProjectName.<directories_between_src_and_enso_file>.<file_without_ext>`
```

Comment:
We could consider adding link to the long story: https://dev.enso.org/docs/distribution/packaging.html
However I'm not sure, because such links like to be invalidated.